python/pyarrow/tests/test_dataset.py

```diff
@@ -2725,6 +2725,16 @@ def test_open_dataset_from_uri_s3(s3_example_simple, dataset_reader):
     assert dataset_reader.to_table(dataset).equals(table)
 
 
+@pytest.mark.parquet
+@pytest.mark.s3
+def test_dataset_from_fileinfos(s3_example_simple, dataset_reader):
```

Comment:
```suggestion
def test_open_dataset_from_fileinfos(s3_example_simple, dataset_reader):
```

(just to keep the naming consistent with the surrounding functions)